methods/media/getMediaMediaName.md

```diff
@@ -7,6 +7,12 @@ Downloads a media file you previously uploaded.
 
 <code class="get">GET</code>`https://api.catapult.inetwork.com/v1/users/{userId}/media/{mediaName}`
 
+## ⚠️ Caution on fetching Media ⚠️
+
+You **MUST** use your API token and secret to display the media each and every time you want to access the file.  We **DO NOT** recommend using Bandwidth's url to display/stream media files to your end-users.  Providing your user-id, token, and secret to users' devices is a security risk, as they _could_ use your credentials to access your account.
```

Comment:
"secret to display" -> "secret to download"

Reply:
Thank you